pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -262,6 +263,89 @@ func (b *OFBridge) IsConnected() bool {
 	return b.ofSwitch.IsReady()
 }
 
+func (b *OFBridge) AddFlowsInBundle(addflows []Flow, modFlows []Flow, delFlows []Flow) error {
+	// If none Openflow entries are requested to be added or modified or deleted on OVS, return.
+	if addflows == nil && modFlows == nil && delFlows == nil {
```

Comment:
is it worth checking if the length of the slice is 0 (for the non-nil case) as well?

Reply:
Thanks for catching it, I would change to use `len()` to check if there is no items in the slice, as len(slice) == 0 if it is nil.